aten/src/ATen/native/AutogradComposite.cpp

```diff
@@ -8,7 +8,8 @@ namespace native {
 // because codegen assumes that a function of this name exists in the native namespace
 // TODO: check the behavior of this and _fw_primal in inference mode
 Tensor _make_dual(const Tensor& primal, const Tensor& tangnet, int64_t level) {
-  AT_ERROR("This is just a stub.");
+  TORCH_INTERNAL_ASSERT("This is just a stub.");
```

Comment:
I think you need a boolean: `TORCH_INTERNAL_ASSERT(false, "This is just a stub")`. Otherwise the string evaluates to True